app/views/simple_smart_answers/_node.html.erb

```diff
@@ -4,10 +4,14 @@
     <%= f.link_to_remove "<i class=\"glyphicon glyphicon-remove glyphicon-smaller-than-text add-right-margin\"></i> Remove node".html_safe, :class => "btn btn-default pull-right remove-node" %>
   </legend>
   <%= f.hidden_field :order, class: 'node-order' %>
+
   <div class="form-group">
-    <span class="form-wrapper">
+    <div class="form-wrapper">
       <%= f.text_field :title, placeholder: "The title of the outcome", class: "node-title form-control" %>
-    </span>
+    </div>
+    <% if f.object.errors.has_key?(:title) %>
+      <%= form_errors(f.object.errors[:title]) %>
+    <% end %>
```

Comment:
```suggestion
    <%= form_errors(f.object.errors[:title]) %>
```

It looks like [we have a convention already](https://github.com/alphagov/publisher/blob/a6438219a710f8dc6ddb23fd59eb9db0f7c71000/app/helpers/form_helper.rb#L18) to call this helper whether there are errors or not.

Reply:
Same thing on the other two instances.

Reply:
Yes, sorry.  Fixed